### ISSUE TITLE
Make the id column for region matching configurable

### ISSUE THREAD

**Issue by andreash0** [hoga-it/leafdown#18]
Until now, it is required to have a "GID_1" column in the upper and lower spdf.
However, we can do this more flexible by having a `gid_column` argument when we create a new `leafdown` object.

```r
Leafdown$new(spdfs_list, map_output_id = "leafdown", input = input, gid_column = "GID_1")
```

**Comment by andreash0** [comment]
... update documentation & intro article afterwards.

**Comment by andreash0** [comment]
Closed with PR #19
